home/content_import_export.py

```diff
@@ -770,8 +774,39 @@ def from_platform_body_element(
             if "next_prompt" in platform_body_element.value
             else None
         )
+        message.buttons = (
+            cls.serialise_buttons(platform_body_element.value["buttons"])
+            if "buttons" in platform_body_element.value
+            else ""
+        )
         return message
 
+    @classmethod
+    def serialise_buttons(cls, buttons: blocks.StreamValue.StreamChild) -> str:
```

Comment:
Out of scope for this PR, but should we perhaps copy and refactor the exporter like we did for the importer?